File: src/api/Model/StatisticsByMonth.cs
using Newtonsoft.Json;

namespace PhoneStoreManager.Model
{
    public class StatisticsByMonth
    {
        [JsonProperty("year")]
        public int? YearFrom { get; set; } = null;

        [JsonProperty("month")]
        public int? MonthFrom { get; set; } = null;

        [JsonIgnore]
        public int? YearTo { get; set; } = null;

        [JsonIgnore]
        public int? MonthTo { get; set; } = null;

        [JsonProperty("totalprice")]
        public double TotalPrice { get; set; } = 0.0;

        [JsonProperty("totaldelivery")]
        public int TotalDelivery { get; set; } = 0;

        [JsonProperty("completeddeliery")]
        public int CompletedDelivery { get; set; } = 0;

        [JsonProperty("completedpercent")]
        public double CompletedPercent { get; set; } = 0.0;

        [JsonProperty("maxpricedeliveryid")]
        public int? MaxPriceDeliveryID { get; set; } = null;

        #region
        [JsonProperty("maxpricedelivery")]
        public BillSummary? BillSummary { get; set; } = null;
        #endregion
    }
}

File: src/web/src/main/java/io/zoemeow/pbl6/phonestoremanager/model/bean/StatisticsByMonth.java
package io.zoemeow.pbl6.phonestoremanager.model.bean;

import com.google.gson.annotations.SerializedName;

public class StatisticsByMonth {
    @SerializedName("year")
    Integer yearFrom;

    @SerializedName("month")
    Integer monthFrom;

    @SerializedName("totalprice")
    Double totalPrice;

    @SerializedName("totaldelivery")
    Integer totalDelivery;

    @SerializedName("completeddeliery")
    Integer completedDelivery;

    @SerializedName("completedpercent")
    Double completedPercent;

    @SerializedName("maxpricedeliveryid")
    Integer maxPriceDeliveryId;

    @SerializedName("maxpricedelivery")
    BillSummary maxPriceDelivery;

    public Integer getYearFrom() {
        return yearFrom;
    }
    public void setYearFrom(Integer yearFrom) {
        this.yearFrom = yearFrom;
    }
    public Integer getMonthFrom() {
        return monthFrom;
    }
    public void setMonthFrom(Integer monthFrom) {
        this.monthFrom = monthFrom;
    }
    public Double getTotalPrice() {
        return totalPrice;
    }
    public void setTotalPrice(Double totalPrice) {
        this.totalPrice = totalPrice;
    }
    public Integer getTotalDelivery() {
        return totalDelivery;
    }
    public void setTotalDelivery(Integer totalDelivery) {
        this.totalDelivery = totalDelivery;
    }
    public Integer getCompletedDelivery() {
        return completedDelivery;
    }
    public void setCompletedDelivery(Integer completedDelivery) {
        this.completedDelivery = completedDelivery;
    }
    public Double getCompletedPercent() {
        return completedPercent;
    }
    public void setCompletedPercent(Double completedPercent) {
        this.completedPercent = completedPercent;
    }
    public Integer getMaxPriceDeliveryId() {
        return maxPriceDeliveryId;
    }
    public void setMaxPriceDeliveryId(Integer maxPriceDeliveryId) {
        this.maxPriceDeliveryId = maxPriceDeliveryId;
    }
    public BillSummary getMaxPriceDelivery() {
        return maxPriceDelivery;
    }
    public void setMaxPriceDelivery(BillSummary maxPriceDelivery) {
        this.maxPriceDelivery = maxPriceDelivery;
    }

    
}
